.github/workflows/docker-publish.yml

```
@@ -0,0 +1,49 @@
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  docker-build-and-push:
+    runs-on: ubuntu-20.04-4-cores
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)